g4f/Provider/Vercel.py

```diff
@@ -1,63 +1,72 @@
 from __future__ import annotations
 
-import base64
-import json
-import uuid
+import base64, json, uuid, quickjs, random
+from curl_cffi.requests import AsyncSession
 
-import quickjs
-from curl_cffi import requests
+from ..typing       import Any, TypedDict
+from .base_provider import AsyncProvider
 
-from ..typing import Any, CreateResult, TypedDict
-from .base_provider import BaseProvider
 
-
-class Vercel(BaseProvider):
-    url                   = "https://play.vercel.ai"
+class Vercel(AsyncProvider):
+    url                   = "https://sdk.vercel.ai"
     working               = True
     supports_gpt_35_turbo = True
+    model                 = "replicate:replicate/llama-2-70b-chat"
```

Comment:
Not in `models.py`

Reply:
I a added "g4f.models.default" for HuggingChat and OpenAssistant.